packages/p2p/src/components/modal-manager/modals/ad-visibility-error-modal/ad-visibility-error-modal.tsx

```diff
@@ -60,29 +53,31 @@ const AdVisibilityErrorModal = ({ error_code }) => {
                                 />,
                             ]}
                         />
-                    </Text>
-                </Modal.Body>
-                <Modal.Footer>
-                    <Button has_effect text={localize('Ok')} onClick={() => hideModal()} primary large />
-                </Modal.Footer>
-            </Modal>
-        );
-    }
+                    ),
+                };
+            default:
+                return {
+                    title: localize("Something's not right"),
```

Comment:
have a doubt.. didn't we change the default error message to 'sorry for the interruption' title and description as 'our servers hit a bump. let's refresh...'? when i initially added this error modal it didn't have a design for the default error message..

Reply:
Hmm not sure about that? isn't that message for unhandled error modal?

Reply:
i'm not sure.. in this case now we have the same title and description.. but to be honest, i dont think we will get the default case since we call this error modal only when the error code for either of the two cases are satisfied.

Reply:
Yeah i checked the design, there is no case for other error messages. So, i guess the modal will not get to that case. Or should we define the type of error_code to only those 2?